src/main.js

```diff
@@ -9,7 +9,7 @@ var Menu = electron.Menu;
 var globalShortcut = electron.globalShortcut;
 var BrowserWindow = electron.BrowserWindow;
 
-const settings = require('electron-settings');
+var settings = require('electron-settings');
```

Comment:
Yea, I haven't investigated why the main electron app wrapper won't work with ES6